var/spack/repos/builtin/packages/allinea-forge/package.py

```diff
@@ -48,5 +48,5 @@ def url_for_version(self, version):
         return url + "allinea-forge-%s-Redhat-6.0-x86_64.tar" % version
 
     def install(self, spec, prefix):
-        textinstall = which('textinstall.sh')
+        textinstall = which(join_path(self.stage.source_path, 'textinstall.sh'))
```

Comment:
Adding `./` should also work. Probably better to use `Executable` instead of `which` since we aren't searching `PATH`, we know exactly where it is already.

Reply:
Ah yes! done!